topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -362,7 +372,7 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >    > >   pre_tasks:
 >    > >     - name: Install Dependencies
 >    > >       package:
->    > >         name: ['git', 'make', 'python-psycopg2', 'python-virtualenv']
+>    > >         name: ['git', 'make', 'python3-psycopg2', 'python3-virtualenv']
```

Comment:
So if you get this error, it's because python2 is available on the system. Ansible mistakenly uses it in some places when it's there. So this is due to a py2-contaminated VM. If you purge any py2 packages, this should work again for you with `python3-pscopg2`

```suggestion
>    > >         name: ['git', 'make', 'python3-psycopg2', 'virtualenv']
```

Reply:
I solved this for good in commit b1953f01f20abc939e9053e56c74050a8e561405 :tada: